### PR TITLE
[zh-cn]sync patch-releases network-policies

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/concepts/services-networking/network-policies.md
content/zh-cn/releases/patch-releases.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/services-networking/network-policies.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ allowed connections will also be implicitly allowed.
 The effects of those `egress` lists combine additively.
 -->
 默认情况下，一个 Pod 的出口是非隔离的，即所有外向连接都是被允许的。如果有任何的 NetworkPolicy
-选择该 Pod 并在其 `policyTypes` 中包含 “Egress”，则该 Pod 是出口隔离的，
+选择该 Pod 并在其 `policyTypes` 中包含 "Egress"，则该 Pod 是出口隔离的，
```
Comment (howieyuen) [inline content/zh-cn/docs/concepts/services-networking/network-policies.md:115]: why use English quotation instead of Chinese? same as below

Comment (my-git9) [reply content/zh-cn/docs/concepts/services-networking/network-policies.md:115]: The characters in the quotation marks are in English. From preview, I think the English quotes seems better.
image
image
What do you think?

Comment (howieyuen) [reply content/zh-cn/docs/concepts/services-networking/network-policies.md:115]: cc @tengqm

Comment (tengqm) [reply content/zh-cn/docs/concepts/services-networking/network-policies.md:115]: I don't have a strong opinion on this.

**Review by tengqm — COMMENTED** [review]
/approve

**Pull request merged**